### DISCUSSION TITLE
Updated SDNext, Error on Attempt to Generate: "cannot import name 'ReturnedEmbeddingsType' from 'compel'"

### DISCUSSION THREAD

**Discussion by fcabanski** [vladmandic/sdnext#2007] [General]
After the latest update (11:29AM 8/14), GENERATE results in an error: cannot import name 'ReturnedEmbeddingsType' from 'compel' (C:\Users\Frank\Vlad\automatic\venv\lib\site-packages\compel__init__.py)

Any ideas?

**Comment by vladmandic** [comment]
uninstall compel so correct version can be installed by installer.
right now, required version of compel doesn't have a version string so its impossible for installer to check if correct version is installed.
as soon as author releases a new version, installer will deal with this automatically.

**Reply by fcabanski** [reply]
Thanks. That fixed it.

Deleted the compel dir and the compel dir that showed the version number. Mentioning that in case someone else has the issue and, like I didn't, doesn't know how to uninstall compel.

**Reply by vladmandic** [reply]
ah, sorry, it would be:
1. activate `venv`:
- on linux: `source venv/bin/activate`
- on windows: `venv/Scripts/activate`
2. uninstall: `pip uninstall compel`

**Reply by strouder** [reply]
it should be .venv/Scripts/activate ... not venv/Scripts/activate it needs to have a dot in the beginning...

**Comment by flashbackpoductions** [comment]
thank you. it helped me to
